app/assets/stylesheets/article-show.scss

```diff
@@ -1198,7 +1138,7 @@ article {
             margin-left: 15px;
           }
 
-          input[type="checkbox"] {
+          input[type='checkbox'] {
```

Comment:
Any particular reason for using `'` over `"` ? Asking out of curiosity..

Reply:
Once again prettier formatting. Pull it locally to try the branch.